src/js/components/tasks/submit.tsx

```diff
@@ -48,6 +50,27 @@ const SubmitModal = ({
     }
   };
 
+  let obejctType;
+  switch (instanceType) {
+    case 'task':
+      objectType = {
+        /* eslint-enable @typescript-eslint/camelcase */
+        objectType: OBJECT_TYPES.TASK_PR,
+        url: window.api_urls.task_create_pr(instanceId),
+      };
+      break;
+    case 'project':
+      objectType = {
+        /* eslint-enable @typescript-eslint/camelcase */
+        objectType: OBJECT_TYPES.PROJECT_PR,
+        url: window.api_urls.project_create_pr(projectId),
+      };
+      break;
+    default:
+      console.warning('Invalid instance type for SubmitModal');
+      return null;
```

Comment:
@wlonk `useForm` just needs to be called before any `return` statement. You can just set a variable here or something, then `return null` after `useForm`. But really -- this is not a possible use-case, so I'm not sure we need to worry about handling it.